rust/src/storage/azure/mod.rs

```diff
@@ -22,6 +22,7 @@
 
 use azure_core::HttpError as AzureError;
 use azure_storage::blob::prelude::*;
+// use azure_storage::data_lake::prelude::*;
```

Comment:
Rather than comment this out, can this just be removed?

Reply:
When I promote it from draft PR, `azure_storage::blob` will be gone and we'll only use `azure_storage::data_lake`.